app/src/main/kotlin/com/wire/android/ui/userprofile/common/UserProfileInfo.kt

```diff
@@ -231,6 +240,16 @@ fun UserProfileInfo(
     }
 }
 
+@SuppressLint("ComposeNamingLowercase")
```

Comment:
Is this being used? 🤔 

As the composable function starts with an Uppercase letter, or it wants something like: `UserName`? 
just curious

Reply:
Good catch, probably a leftover